### PR TITLE
Add scatter plot functionality

### PR DESCRIPTION
**Pull request by avonow**
Adds the option to create a scatter plot.

### CODE REVIEW

**Review by huntc — COMMENTED** [review]
Thanks for this!

Would it be possible to provide an example of scatter in use (see how examples are structured…)?

File: src/horizontal_series.rs [hunk 1]
```diff
@@ -119,17 +120,26 @@ impl HorizontalSeries {
                         svg_elements.push(
                             html!(<polyline points={points} class={classes.to_owned()} fill="none"/>),
                         );
-                    } else {
+                    } else if props.series_type == SeriesType::Bar {
```
Comment (huntc) [inline src/horizontal_series.rs]: Cargo clippy should probably highlight that we need a match statement now…

Comment (avonow) [reply src/horizontal_series.rs]: Updated to use match statements instead. (Though, Cargo clippy didn't highlight the need for a match; Rory also found this on the code.)

**Comment by avonow** [comment]
In the last commit here, I've added images from my scatter implementation and updated the README. I changed the layout slightly, so that the images will be at 70% and centred. If this is not desirable, very happy to revert.

**Comment by avonow** [comment]
We've been informed that the `data_labels` property can be used for scatter plots, so a separate PR will be raised to just document that with an example, instead of this current PR.

**Comment by huntc** [comment]
> We've been informed that the `data_labels` property can be used for scatter plots, so a separate PR will be raised to just document that with an example, instead of this current PR.

Thanks so much. That'd be wonderful.

**Pull request merged**